python/mach_bootstrap.py

```diff
@@ -184,8 +167,8 @@ def _activate_virtualenv(topdir, is_firefox):
     # and it will check for conflicts.
     requirements_paths = [
         os.path.join("python", "requirements.txt"),
-        wptrunner_path(is_firefox, topdir, "requirements.txt",),
-        wptrunner_path(is_firefox, topdir, "requirements_firefox.txt"),
+        os.path.join(WPT_RUNNER_PATH, "requirements.txt",),
+        os.path.join(WPT_RUNNER_PATH, "requirements_firefox.txt"),
```

Comment:
Is `requirements_firefox.txt` still needed?

Reply:
Great question. I've kicked off a try job, attempting to remove that. It's hard to say which requirements inside of the Servo WPT executor might be accidentally relying on Gecko's list of packages.